src/main/java/vivatech/util/TierHelper.java

```diff
@@ -0,0 +1,14 @@
+package vivatech.util;
+
+import net.minecraft.item.Item;
+import net.minecraft.util.Identifier;
+import net.minecraft.util.registry.Registry;
+
+public class TierHelper {
+	
+	public static void registerTeiredItems(Item[] itemArray, Identifier id) {
+		for (int i = 0; MachineTiers.values().length < i; i++) {
+			Registry.register(Registry.ITEM, new Identifier(id.getNamespace(), id.getPath() + MachineTiers.values()[i]), itemArray[i]);
```

Comment:
Will this produce the most natural sounding machine identifiers? For instance, with an identifier of `electric_furnace` and a tier of `_minimal` this will make `electric_furnace_minimal`, which, aside from stuffing the underscore into the machine identifiers which is a bad idea, screws up the english word order. I suggest:
```java
new Identifier(id.getNamespace(), MachineTiers.values()[i] + "_" + id.getPath())
```
and dropping the underscores from MachineTiers, resulting in machine id's like `minimal_electric_furnace`.

Reply:
Neat idea!

Reply:
Yeah, that a cleaner way to put it.